docs: add warning (#642)
bytebase/bytebase.com: content/docs/administration/webhook-integration/external-approval.md

![feishu app permission page](/static/docs/external-approval-feishu-permission.webp)

> <hint-block type="warn">
> 
> Make sure that you have added all three scopes before moving on.
> 
> The "approval:approval" scope may seem to contain "approval:approval:readonly", but you have to add both.
> 
> </hint-block>

Reviewer: ```suggestion
<hint-block type="warn">

Make sure that you have added all three scopes before moving on. In particular, you must add both "approval:approval" and "approval:approval:readonly".

</hint-block>
```
